File: LeetCode/Csharp/Easy/TwoSum.cs

partial class Program
{
    public int[] TwoSum(int[] nums, int target)
    {
        int[] vals = new int[2];
        int arrayLength = nums.Length;

        for(int i = 0; i < arrayLength; i++)
        {
            for(int j = i + 1; j < arrayLength; j++)
            {
                if (nums[i] + nums[j] == target)
                {
                    vals[0] = i;
                    vals[1] = j;
                    return vals;
                }
            }
        }
        return vals;
    }
}

File: LeetCode/Cpp/Easy/TwoSum.cpp
#include "Header.h";

vector<int> twoSum( vector<int>& nums, int target) {
    vector<int> newVect;
    for (int i = 0; i < nums.size(); i++) {
        for (int j = i + 1; j < nums.size(); j++) {
            if (nums.at(i) + nums.at(j) == target) {
                newVect.push_back(i);
                newVect.push_back(j);
                break;
            }
        }
    }
    return newVect;
}